mediathread/assetmgr/models.py

```diff
@@ -339,3 +339,22 @@ def __unicode__(self):
 
     class Meta:
         ordering = ['title']
+
+
+class JuxtaposeAsset(models.Model):
+    added = models.DateTimeField(auto_now_add=True)
+    modified = models.DateTimeField(auto_now=True)
+    author = models.ForeignKey(User)
+    course = models.ForeignKey(Course)
+    title = models.TextField(blank=True, null=True)
```

Comment:
I was thinking of the title as equivalent to the Juxtaposition's "global annotation", which is why I had it optional. I will make this required.